common/info.yaml

```
@@ -1292,6 +1292,128 @@ description: |
       Number and value of presets undefined and may vary by model.
       Useful for very basic integrations where you have just a single push button.
 
+  ## Color
+
+  The Color feature is available on full color lighting products.
+
+  The Color feature requires the Light and Brightness features.
+  The ColorTemp feature is usually not available at the same time as Color.
+
+  When `color = 1`, the values in `rgb` and `hsv` 
+  are active and specify the color of light from the device.
+
+  Whenever `SetBrightness` or other brightness-changing actions are called,
+  `rgb` and `hsv` values are updated to reflect the change. For example, if the 
+  rgb values are (128, 0, 0) with brightness 50, and brightness is increased
+  to 100, then rgb values are updated to (255, 0, 0).
+
+  **Note regarding the value (`v`) component of HSV colorspace:**
+  An RGB (or RGBW) LED has an inherent range of colors which it can produce.
+  This is known as the device's *gamut*. In RGB- (or RGBW-) space the shape
+  of the gamut is a cube (or hypercube). However, when this gamut is transformed
+  into HSV space, it becomes tent-shaped (a cylinder with a cone on top).
+  Intitively, what is going on is that when producing a pure color, 
+  say pure red, the device has only a single physical LED channel to contribute 
+  to the overall physical brightness (value) of the light. The brightest
+  pure colors are along the top of the tent wall (the top circumferance of
+  the cylinder). However, when generating white light, the device can bring 
+  to bare all three (or four) channels and achieve a much higher value.
+  This is the very top of the tent (apex of the cone).
+
+  ### Properties
+
+    (none)
+
+  ### State Variables
+
+    - **color**: (integer) 0=white, 1=color in `rgb` active.
+    - **rgb**: (object) JSON object containing the following keys
+       which describe the apparent color of the light.
+       * `r` - (0--255) apparent red value
+       * `g` - (0--255) apparent green value
+       * `b` - (0--255) apparent blue value
+    - **hsv**: (object) similar to `rgb` but for the HSV colorspace.
+       * `h` - (0--359) apparent hue (in degrees)
+       * `s` - (0--1000) apparent saturation (in permilles)
+       * `v` - (0--1000) apparent value in HSV colorspace (in permilles).
+               Note that `v` and `brightness` are not the same.
+               Users should typically use `brightness` rather than `v`.
+
+  ### Actions
+
+    - **SetRGB({r:int, g:int, b:int})**
+      Set color by RGB values.
+      Light is implicitely turned on if off.
```

Typo?
```suggestion
      Light is implicitly turned on if off.
```

(There are other occurrences)